From e59c5514349f897133f277bdafd67f19742545e8 Mon Sep 17 00:00:00 2001
From: Sam Ravnborg <srn@skov.dk>
Date: Sat, 14 Jan 2017 21:50:46 +0100
Subject: [PATCH] decrease debug level to see less noise during boot

The following is logged during boot:

SD/MMC: Image: Read file u-boot.bin to 0x23800000
Cmd: 0x1 Response Time-out
SD: Card Capacity: Standard
SD: Specification Version 2.00
Cmd: 0xd, error check, status: 0x4c0e5
Cmd: 0xd, error check, status: 0x4c0e5
Cmd: 0xd, error check, status: 0x4c0e5
Cmd: 0xd, error check, status: 0x4c0e5
<repeats many times>
SD/MMC: Done to load image

The reason for the time-out is not know.
Likewise the reason for the error check that fails is unknown.

With this we simply close our eyes and pray this is something
we can ignore.
---
 contrib/board/skov/arm9_cpu/arm9_cpu_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
index 19c9070..2b13541 100644
--- a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
+++ b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
@@ -4,4 +4,3 @@ CONFIG_SDRAM_16BIT=y
 CONFIG_SDCARD=y
 CONFIG_JUMP_ADDR="0x23800000"
 CONFIG_DEBUG=y
-CONFIG_DEBUG_LOUD=y
-- 
1.8.3.1

